src/MultiUserEditingController.php

```diff
@@ -75,7 +69,7 @@ public function set($request)
 
         $dataArray = array();
         $dataArray['lastEdited'] = date("Y-m-d H:i:s"); //when this was last updated
-        $dataArray['abbreviatedName'] = $this->user->FirstName . ' ' . substr($this->user->Surname, 0, 1);
+        $dataArray['abbreviatedName'] = $this->user->FirstName . ' ' . ($this->user->Surname ? substr($this->user->Surname, 0, 1) : '');
```

Comment:
```suggestion
        $dataArray['abbreviatedName'] = $this->user->FirstName . ' ' . substr($this->user->Surname ?: '', 0, 1);
```

Reply:
Of course. That is much cleaner. Thanks. I have made this change.